lib/teams/server/message-actions/chat-message-actions.js

```diff
@@ -0,0 +1,235 @@
+const { sc, Logger } = require('@reldens/utils');
+const { ChatConst } = require('../../../chat/constants');
+const { TeamsConst } = require('../../constants');
+
+class ChatMessageActions
+{
+    constructor(props)
+    {
+        this.events = sc.get(props, 'events', false);
+        this.chatPlugin = sc.get(props, 'chatPlugin', false);
+        this.client = false;
+        if (!this.events) {
+            Logger.error('EventsManager undefined in Teams ChatMessageActions');
+        }
+        if (!this.chatPlugin) {
+            Logger.error('ChatPlugin undefined in Teams ChatMessageActions');
+        }
+    }
+
+    listenEvents()
+    {
+        this.inviteTeamAcceptedEventListener();
+        this.inviteTeamRejectedEventListener();
+        this.teamMemberLeaveEventListener();
+        this.inviteClanAcceptedEventListener();
+        this.inviteClanRejectedEventListener();
+        this.clanMemberLeavingEventListener();
+    }
+
+    inviteTeamAcceptedEventListener()
+    {
+        this.events.on('reldens.afterPlayerJoinedTeam', (props) => {
+            let playerJoiningSchema = sc.get(props, 'playerJoiningSchema', false);
+            if (!playerJoiningSchema) {
+                return false;
```

Comment:
log warnings here since the properties are expected to be available on the events